src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -1904,8 +1905,11 @@ private String init(boolean initFull) {
                         }
                     } catch (RuntimeException ex) {
                         logger.warning("Problem getting rsync script: " + ex.getLocalizedMessage());
+                        FacesContext.getCurrentInstance().addMessage("Dataset Page Runtime", new FacesMessage("message")
```

Comment:
Is "Dataset Page Runtime" really what we want to show the user when there's a RuntimeException? Maybe "Dataset Page RuntimeException" instead? And perhaps ex.getLocalizedMessage too? Or "Failed to get rsync script."?

Reply:
Right. Thanks. I moved the Exception Type to the logger for diagnostics. The user doesn't need to see it.